jupyter-extension/jupyterlab_pachyderm/handlers.py

```diff
@@ -331,7 +331,8 @@ async def put(self):
             address = body["pachd_address"]
             cas = bytes(body["server_cas"], "utf-8") if "server_cas" in body else None
 
-            if address.removeprefix("grpc://") != self.client.address or cas:
+            # TODO: Can this conditional be removed entirely?
+            if address.endswith(self.client.address) or cas:
```

Comment:
I think removing the conditional entirely is preferable, it should be fine to just re-establish the client in all cases.